### ISSUE TITLE
Add new rule `svelte/prefer-interpolation-at-attributes`

### ISSUE THREAD

**Issue by oekazuma** [sveltejs/eslint-plugin-svelte#1457]
### Motivation

Currently, there is no rule to prevent the redundant use of JavaScript template literals within Svelte attributes.

While `<div class={`${base} extra`}>` is valid, Svelte's compiler is specifically designed to handle direct interpolation like `<div class="{base} extra">`. The latter is the idiomatic Svelte way, which offers several advantages:

1. **Syntax Noise**: Template literals introduce unnecessary backticks (```) and dollar signs (`$`) inside a template that already uses curly braces for reactivity. This creates visual clutter.
2. **Framework Consistency**: Svelte's documentation and community standards favor using attribute quotes with internal mustache tags for string concatenation.
3. **Complexity Oversight**: Developers coming from React (JSX) often carry over the habit of using template literals for everything. This rule would help them transition to Svelte's more concise syntax.

The goal is to enforce a cleaner template structure by flagging cases where a template literal is the sole content of an attribute tag and can be simplified into Svelte's native interpolation.

### Description

This rule enforces the use of Svelte's native attribute interpolation instead of using JavaScript template literals inside a mustache tag.

Specifically, it flags cases where an attribute value consists entirely of a single mustache tag containing a template literal (e.g., attr={${val} text}). The rule will suggest converting these to Svelte's standard format (e.g., attr="{val} text").

This improves template scannability by removing redundant syntax characters (backticks and $) and aligns the code with Svelte's idiomatic style for attribute value concatenation.

### Examples

```svelte
<!-- ✓ GOOD -->
<div class="{prefix} container" />
<img alt="{dynamicValue} image" />
<button data-type="btn-{isVisible ? 'primary' : 'secondary'}" />

<!-- ✗ BAD -->
<div class={`${prefix} container`} />
<img alt={`${dynamicValue} image`} />
<button data-type={`btn-${isVisible ? 'primary' : 'secondary'}`} />
```

**Comment by sacrosanctic** [comment]
```svelte
<div class={[prefix, "container"]}></div>
<img alt={[dynamicValue, "image"]}></div>
```
With the introduction of `clsx`, this is more idiomatic. Quotes are wrapped around actual strings; there is no need to keep track of whitespaces and is more readable as the number of static and dynamic variables increases.

**Comment by balthild** [comment]
`clsx` does not work in many senarios. For example:

```html
<img src="https://cdn.jsdelivr.net/.../{id}.png" />
<svg><symbol id="star-hard-{uniqueId}">...</symbol></svg>
<input placeholder="{urlBase}?userId=..." />
```

IMHO this lint is very useful and should be enabled by default. I didn't even realize until just now that Svelte supports string interpolation within attributes -- this wouldn't happen if the lint exists.
